docs/Get-HelpPreview.md

```diff
@@ -34,44 +34,43 @@ Update-MarkdownHelpModule
 Update-MarkdownHelpSchema
 ```
 
-Returns a help object get-help preview from maml xml and assign it to the $help variable.
-Gets the names of Cmdlet objects inside help.
+The first command creates a **Help** object for the the specified MAML file. The command stores it in the $Help variable.
+
+The second command displays the **Name** property for each of the objects in $Help.
+
 ## PARAMETERS
 
 ### -Path
-Path to MAML help files.
-You can pass several of them.
+Specifies an array of paths of MAML external help files.
 
 ```yaml
 Type: String[]
 Parameter Sets: (All)
-Aliases: 
+Aliases:
 
 Required: True
 Position: Named
-Default value: 
+Default value:
 Accept pipeline input: True (ByValue)
 Accept wildcard characters: False
 ```
 
-### CommonParameters
```

Comment:
I see that you alternated the link in other cases, but in this one you just removed the whole entry.
Perhaps, it was the tool who removed it. Did you use `Update-MarkdownHelp` ?